stacks/play.prx.org.yaml

```diff
@@ -0,0 +1,238 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: play.prx.org application running in Docker
+Mappings:
+  Shared:
+    Project:
+      name: play.prx.org
+    Container:
+      name: play-express
+Conditions:
+  CreateStagingResources: !Equals [!Ref EnvironmentType, Staging]
+  CreateProductionResources: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  # VPC ########################################################################
+  VPC:
+    Type: "AWS::EC2::VPC::Id"
+  # Load Balancer ##############################################################
+  PlatformALBDNSName:
+    Type: String
+  PlatformALBCanonicalHostedZoneID:
+    Type: String
+  PlatformALBHTTPListenerArn:
+    Type: String
+  PlatformALBHTTPSListenerArn:
+    Type: String
+  PlatformALBListenerPriorityPrefix:
+    Type: String
+  # ECS Cluster ################################################################
+  ECSCluster:
+    Type: String
+  ECSServiceAutoscaleIAMRoleArn:
+    Type: String
+  ECSServiceIAMRole:
+    Type: String
+  # Misc #######################################################################
+  OpsDebugMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  EcrRegion:
+    Type: String
+  SecretsBase:
+    Type: String
+  SecretsRegion:
+    Type: String
+  # Shared ENV #################################################################
+  # Play ECS ################################################################
+  PlayEcrImageTag:
+    Type: String
+  # Play ENV ################################################################
+  PlaySecretsVersion:
+    Type: String
+Resources:
+  PlayLogGroup:
+    Type: "AWS::Logs::LogGroup"
+    Properties:
+      RetentionInDays: 14
+  PlayALBTargetGroup:
```

Comment:
Should add a deregistration delay here.  (15s is plenty, since this app doesn't really process anything).